xaitk_saliency/impls/perturb_image/rise.py

```diff
@@ -49,6 +49,7 @@ def __init__(
     ):
         """
         Generate a set of random binary masks
+        
```

Comment:
Looks like this newline has to be removed for linting purposes.

Reply:
It's not that there's a new-line here, but that the empty line has trailing white-space. Removing that should resolve the issue.

NOTE: these ghostflow checks about white-space are on a PER-COMMIT basis, so fixing this will require a fixup-commit into the offending commit.